client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.tsx

```diff
@@ -1,22 +1,20 @@
-import { TITAN_MAIL_MONTHLY_SLUG } from '@automattic/calypso-products';
+import { TITAN_MAIL_ANNUALLY_SLUG, TITAN_MAIL_MONTHLY_SLUG } from '@automattic/calypso-products';
 import { Gridicon } from '@automattic/components';
-import formatCurrency from '@automattic/format-currency';
 import { withShoppingCart } from '@automattic/shopping-cart';
-import classNames from 'classnames';
 import { translate } from 'i18n-calypso';
-import page from 'page';
 import React, { FunctionComponent, useState } from 'react';
```

Comment:
Nit: we don't need to import `React` and we should import `FunctionComponent` as a type.